technical-reference.adoc

```diff
@@ -1,512 +1,466 @@
-| limit a| Number | Number of results to return per Page.| 60 |
+| Goal time gte a| DateTime | Filter for Submissions that were created with the `goal-time` parameter whose value is the same or greater than a specific date and time (“greater than or equal to” operator). | |
+| Goal time lt a| DateTime | Filter for Submissions that were created with the goal-time parameter whose value is before a specific date and time (“less than” operator). | |
+| halted a| Boolean | Filters for Submissions that are in a halted state.| |
+| id a| Submission Id | Filters for Submissions based on their `id`s| |
+| layout a| String | Filters for Submissions whose Documents are matched to a certain layout | |
+| layout tag a| String | Filters for Submissions containing Documents that matched to a layout with the specified layout tag.| |
+| limit a| Number | Number of results to return per page.| 60 |
 | offset a| Number | The initial index from which to return the results.| |
-| Start time gte a| DateTime | Filter for Submissions that were ingested into the application on or after a specific date and time (greater than or equal to operator).|  |
-| Start time lt a| DateTime | Filter for Submissions that were ingested into the application before a specific date and time (less than operator).|  |
+| Start time gte a| DateTime | Filter for Submissions that were ingested into the application on or after a specific date and time (“greater than or equal to” operator).|  |
+| Start time lt a| DateTime | Filter for Submissions that were ingested into the application before a specific date and time (“less than” operator).|  |
 | state a| String | Filters for Submissions that are in a specific state. See link:https://docs.hyperscience.com/#substates[Substates] for a list of possible values.| |
-| substate a| String | Filters for Submissions that are in a specific substate. See Substates for a list of possible values.| |
-| Streaming Strategy a| * <<RepeatableInMemoryStream>>
-* <<RepeatableFileStoreStream>>
-* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
-| Custom Query Parameters a| Object | Custom query parameters to include in the request. The specified query parameters are merged with the default query parameters that are specified in the configuration. |  |
-| Custom Headers a| Object | Custom headers to include in the request. The specified custom headers are merged with the default headers that are specified in the configuration. |  |
+| substate a| String | Filters for Submissions that are in a specific substate. See link:https://docs.hyperscience.com/#substates[Substates] for a list of possible values.| |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* <<non-repeatable-stream>> |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
```

Comment:
@rui-lomba , what type of repeatable-stream is the default? We have 2.

Reply:
file-store, added!
Thanks